setup.py

```diff
@@ -41,74 +36,6 @@ def finalize_options(self):
 except ImportError:
     bdist_wheel = None
 
-
-class PostInstallCommand(install):
-    """Post-installation for installation mode."""
-
-    def run(self):
-        source_dir = os.path.dirname(os.path.abspath(__file__))
-        executable_name = "maturin.exe" if sys.platform.startswith("win") else "maturin"
-
-        # Shortcut for development
-        existing_binary = os.path.join(source_dir, "target", "debug", executable_name)
-        if os.path.isfile(existing_binary):
-            source = existing_binary
-        else:
-            # https://github.com/PyO3/maturin/pull/398
-            cargo = shutil.which("cargo") or shutil.which("cargo.exe")
-            if not cargo:
-                raise RuntimeError(
-                    "cargo not found in PATH. Please install rust "
-                    "(https://www.rust-lang.org/tools/install) and try again"
-                )
-
-            cargo_args = [
-                cargo,
-                "rustc",
-                "--release",
-                "--bin",
-                "maturin",
-                "--message-format=json",
-            ]
-
-            if platform.machine() in ("ppc64le", "ppc64", "powerpc") or (
-                sys.platform == "win32" and platform.machine() == "ARM64"
-            ):
-                cargo_args.extend(
-                    ["--no-default-features", "--features=upload,log,human-panic"]
-                )
-            elif sys.platform.startswith("haiku"):
-                # mio and ring doesn't build on haiku
-                cargo_args.extend(
-                    ["--no-default-features", "--features=log,human-panic"]
```

Comment:
We need to keep feature flags, otherwise it won't compile on these platforms.

Reply:
Added those back